conf/node-ready.yaml

```diff
@@ -0,0 +1,15 @@
+# ready is a pseudo boot stage to mark that all
+# zinit unit files provided by the base image
+# have been booted correctly
+
+exec: "true"
+oneshot: true
+after:
+  # list of all services that must be running before moving on
+  # note, these services are defined by the 0-initramfs repo
+  - local-modprobe
+  - udev-trigger
+  - redis
```

Comment:
duplicate from line 12.
Also I wonder if Redis should be there.
Redis is the message broker for zbus. I can change at some point. Shouldn't we move it out of here. Since these are really more hardware setup oriented.

Reply:
that's exactly the reason why it should be part of zos2 not initramfs. because we can just remove it from here (and add the new bus) and push to have a new image.

The upgrader can then take care of updating the config of running machines.